### PR TITLE
#17796 Procedure debugging fix

### LINKED ISSUE #17796: How debug procedure
### Description

Hi.

How I can debug procedure?

I get error now.

“PostgreSQL Debug – Local session 743080
ERROR: recontract.external_load_contract(integer, integer) is a procedure
Подсказка: To call a procedure, use CALL.
Позиция: 15”

Functions are debugged without errors.

Best regards.

### DBeaver Version

Communicate edition 22.2.0

### Operating System

windows 10

### Database and driver

1. Database name and version: PostgreSQL 14.4 on x86_64-pc-linux-gnu
2. Driver name and version: PostgreSQL JDBC Driver 42.2.25

### Steps to reproduce

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by ShadelessFox**
Acceptance criteria:
- [ ] ~~Debugging session can be started against a function~~
- [x] Debugging session can be started against a procedure

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.ext.postgresql/plugin.xml [hunk 1]
```diff
@@ -328,6 +328,7 @@
                     <parameter name="serverType" value="postgresql"/>
                     <property name="loginTimeout" value="20"/>
                     <property name="connectTimeout" value="20"/>
+                    <property name="escapeSyntaxCallMode" value="callIfNoReturn"/>
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ext.postgresql/plugin.xml:331]: So... is debugging available only for Postgres? What about Greenplum? Cockroach? EDB? Redshift? Yellowbrick? etc.

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ext.postgresql/plugin.xml:331]: I believe so. To use the debugger, one must install the native extension for the database.

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.ext.postgresql/plugin.xml:331]: But EDB (as an example) is a kind of clone of Postgres. And supports extensions. Are you sure?

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.ext.postgresql/plugin.xml:331]: I guess it's indeed compatible with EDB (judging by the "about" section https://github.com/EnterpriseDB/pldebugger).

**Review by LonwoLonwo — APPROVED** [review]

**Review by Destrolaric — APPROVED** [review]

**Comment by Matvey16** [comment]
Can't open session for functions

```
org.jkiss.dbeaver.debug.DBGException: Unable to rcv port number
	at org.jkiss.dbeaver.ext.postgresql.debug.internal.impl.PostgreDebugSession.waitPortNumber(PostgreDebugSession.java:312)
	at org.jkiss.dbeaver.ext.postgresql.debug.internal.impl.PostgreDebugSession.attachLocal(PostgreDebugSession.java:404)
	at org.jkiss.dbeaver.ext.postgresql.debug.internal.impl.PostgreDebugSession.attach(PostgreDebugSession.java:479)
	at org.jkiss.dbeaver.ext.postgresql.debug.internal.impl.PostgreDebugController.createSession(PostgreDebugController.java:49)
	at org.jkiss.dbeaver.ext.postgresql.debug.internal.impl.PostgreDebugController.createSession(PostgreDebugController.java:1)
	at org.jkiss.dbeaver.debug.DBGBaseController.openSession(DBGBaseController.java:69)
	at org.jkiss.dbeaver.debug.core.model.DatabaseDebugTarget.connect(DatabaseDebugTarget.java:181)
	at org.jkiss.dbeaver.debug.core.DatabaseLaunchDelegate.launch(DatabaseLaunchDelegate.java:50)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:806)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:717)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1021)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$2.run(DebugUIPlugin.java:1224)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
```

**Comment by ShadelessFox** [comment]
Turns out, the debug extension doesn't support functions. I don't know how it worked before.

**Review by E1izabeth — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**